src/main/java/com/bravo/user/validator/UserValidator.java

```diff
@@ -14,6 +17,20 @@ public void validateId(String id){
       throw new BadRequestException("'id' is required");
     }
   }
+  
+  /**
+   * Validates a user name to ensure it is not null, not empty, and does not contain space, numbers, or special characters.
+   * 
+   * @param name User name to validate
+   * @throws BadRequestException if the name is invalid
+   */
+  public void validateName(String name) {
+	  Pattern pattern = Pattern.compile("[^a-z*%!]", Pattern.CASE_INSENSITIVE);
+	  if (ValidatorUtil.isInvalid(name) || pattern.matcher(name).find()) {
```

Comment:
Why did you use `matcher(name).find()` instead of `matcher(name).matches()`?

Reply:
I did try `matches()`.  It only matches against the full string.  `find()` will also match against substrings, which appeared to work properly when tested.  See:

https://stackoverflow.com/questions/4450045/difference-between-matches-and-find-in-java-regex